lib/src/map/camera/camera.dart

```diff
@@ -208,37 +207,38 @@ class MapCamera {
 
   /// Calculates the size of a bounding box which surrounds a box of size
   /// [nonRotatedSize] which is rotated by [rotation].
-  static Point<double> calculateRotatedSize(
+  static Offset calculateRotatedSize(
     double rotation,
-    Point<double> nonRotatedSize,
+    Offset nonRotatedSize,
   ) {
     if (rotation == 0.0) return nonRotatedSize;
 
     final rotationRad = degrees2Radians * rotation;
     final cosAngle = math.cos(rotationRad).abs();
     final sinAngle = math.sin(rotationRad).abs();
-    final width = (nonRotatedSize.x * cosAngle) + (nonRotatedSize.y * sinAngle);
+    final width =
+        (nonRotatedSize.dx * cosAngle) + (nonRotatedSize.dy * sinAngle);
     final height =
-        (nonRotatedSize.y * cosAngle) + (nonRotatedSize.x * sinAngle);
+        (nonRotatedSize.dy * cosAngle) + (nonRotatedSize.dx * sinAngle);
 
-    return Point<double>(width, height);
+    return Offset(width, height);
   }
 
   /// The current rotation value in radians
   double get rotationRad => rotation * degrees2Radians;
 
   /// Calculates point value for the given [latlng] using this camera's
   /// [crs] and [zoom] (or the provided [zoom]).
-  Point<double> project(LatLng latlng, [double? zoom]) =>
+  Offset project(LatLng latlng, [double? zoom]) =>
       crs.latLngToPoint(latlng, zoom ?? this.zoom);
 
   /// Calculates the [LatLng] for the given [point] using this camera's
   /// [crs] and [zoom] (or the provided [zoom]).
-  LatLng unproject(Point point, [double? zoom]) =>
+  LatLng unproject(Offset point, [double? zoom]) =>
       crs.pointToLatLng(point, zoom ?? this.zoom);
 
   /// Same as the [unproject] function.
-  LatLng layerPointToLatLng(Point point) => unproject(point);
+  LatLng layerPointToLatLng(Offset point) => unproject(point);
```

Comment:
We might want to deprecate this. It's just duplication.

Reply:
yeah it isn't even used